### PR TITLE
Createpoi

### PR DESCRIPTION
**Pull request by Dezedene**
Creation of POI form

### CODE REVIEW

File: src/Components/CreatePoiForm.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    case 'Services Publics':
+      return 3;
+    case 'Vie Quotidienne':
+      return 4;
+    case 'Transports & Mobilité':
+      return 5;
+    default:
+      return keyword;
+  }
+};
```
Comment (erabin05) [inline src/Components/CreatePoiForm.js:30]: Not the right way to get the id of your first level points of interests. You should the ones of your db

File: src/Components/CreatePoiForm.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+          type="text"
+          id="name"
+          name="name"
+          onChange={e => dispatch({ type: 'HANDLE_FORM_NAME_CHANGE', name: e.target.value })}
+          required
+        />
+      </div>
+      <div>
+        <label htmlFor="keywordOne">Catégorie:</label>
+        <br />
```
Comment (erabin05) [inline src/Components/CreatePoiForm.js]: ```<br/>``` !!

File: src/Components/CreatePoiForm.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        <p>
+          Adresse:
+          <span> en fonction des coordonnées gps</span>
+        </p>
+      </div>
+
+
+      <div className="poi-name">
+        <label htmlFor="name">Nom</label>
+        <br />
```
Comment (erabin05) [inline src/Components/CreatePoiForm.js]: ```<br/>``` !!

File: src/Components/CreatePoiForm.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+          {poiKeywordsDisplay.filter(keyword => keyword.importance === 2
+          && getFirstKeywordId(keywordOne) === keyword.parent_id)
+            .map(keyword => (
+              <option key={keyword.name} id={keyword.name} value={keyword.name}>
+                {keyword.name.replace('_', ' ')}
+              </option>
+            ))}
+        </select>
+      </div>
+      <br />
```
Comment (erabin05) [inline src/Components/CreatePoiForm.js]: ```<br/>``` !!

File: src/Components/CreatePoiForm.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+      <div className="poi-pic">
+        <label htmlFor="poiPic">Ajouter une photo</label>
+        <br />
```
Comment (erabin05) [inline src/Components/CreatePoiForm.js]: ```<br/>``` !!

**Review by erabin05 — CHANGES_REQUESTED** [review]
- dont use ```<br/>``` 
- The way you filter the second keyword must change, use the id of your db

File: src/Components/CreatePoiForm.js (hunk 1 above)
Comment (Dezedene) [reply src/Components/CreatePoiForm.js:30]: it's done

**Comment by Dezedene** [comment]
Every <br/> have been removed.

**Comment by Dezedene** [comment]
The way we filter the second keyword has been changed as well.

**Review by erabin05 — CHANGES_REQUESTED** [review]
br !!

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**